app/workers/message_delivery_worker.rb

```diff
@@ -0,0 +1,29 @@
+class MessageDeliveryWorker
+  include Sidekiq::Worker
+
+  class Mailer < ActionMailer::Base
+    default from: 'noreply@madetomeasure.io'
+
+    def message_email(subscriber_id, message_id)
+      sub = Subscriber.find(subscriber_id)
+      msg = Message.find(message_id)
+
+      # FIXME message needs to get rendered with user specific data with templating
```

Comment:
minor observation:  can we add our usernames to code tags? eg:  # NOTE (cmhobbs) flibbity floop